pass-client-integration/src/test/java/org/dataconservancy/pass/client/integration/ClientITBase.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2017 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dataconservancy.pass.client.integration;
+
+import static com.openpojo.reflection.impl.PojoClassFactory.enumerateClassesByExtendingType;
+
+import java.lang.reflect.Method;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.dataconservancy.pass.client.PassClient;
+import org.dataconservancy.pass.client.fedora.FedoraPassClient;
+import org.dataconservancy.pass.model.PassEntity;
+
+import org.apache.commons.beanutils.BeanUtils;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import com.openpojo.reflection.PojoClass;
+
+/**
+ * Base class for client ITs.
+ *
+ * @author apb@jhu.edu
+ */
+public abstract class ClientITBase {
+
+    static {
+        if (System.getProperty("pass.fedora.baseurl") == null) {
+            System.setProperty("pass.fedora.baseurl", "http://localhost:8080/fcrepo/rest/");
+        }
+    }
+
+    /** All PassEntity classes from pass-model */
+    @SuppressWarnings("unchecked")
+    protected static final Collection<Class<? extends PassEntity>> PASS_TYPES = enumerateClassesByExtendingType(
+            "org.dataconservancy.pass.model", PassEntity.class, null)
+                    .stream()
+                    .map(PojoClass::getClazz)
+                    .map(cls -> (Class<? extends PassEntity>) cls).collect(Collectors.toList());
+
```

Comment:
OK, I tried this out. I did not need to change the package name, it was recursive, but I did need to add this dependency to integration. 
```
    <dependency>
      <groupId>org.dataconservancy.pass</groupId>
      <artifactId>pass-model-nihms</artifactId>
      <version>${project.version}</version>
    </dependency>
```
That also revealed the need for this line in the embedded context:
```
    "pmid": {"@id": "pass:pmid"},
```
Once those two things are in place this works.

Reply:
That makes sense - yes if it's in a different maven module, then it would need to be pulled in as a dependency.  I'll make those changes and update the PR